server/services/v1/query_runner.go

```diff
@@ -566,11 +583,23 @@ func (runner *SearchQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 
 		var row Row
 		for rowReader.Next(ctx, &row) {
+			var createdAt, updatedAt *timestamppb.Timestamp
+			if row.Data.CreatedAt != nil {
```

Comment:
row.Data could have helpers GetCreatedAt(), GetUpdatedAt() so we don't need to check everytime and use it directly below.